kotlin-js/src/commonMain/kotlin/js/core/JsNumber.kt

```diff
@@ -33,3 +47,57 @@ fun Double.toExponential(): String =
 
 fun Double.toExponential(fractionDigits: Int): String =
     JsNumber(this).toExponential(fractionDigits)
+
+fun Int.toFixed(): String =
+    JsNumber(this).toFixed()
+
+fun Int.toFixed(digits: Int): String =
+    JsNumber(this).toFixed(digits)
+
+fun Double.toFixed(): String =
+    JsNumber(this).toFixed()
+
+fun Double.toFixed(digits: Int): String =
+    JsNumber(this).toFixed(digits)
+
+fun Int.toLocaleString(): String =
+    JsNumber(this).toLocaleString()
+
+fun Int.toLocaleString(locales: String): String =
+    JsNumber(this).toLocaleString(locales)
+
+fun Int.toLocaleString(locales: String, options: NumberFormatOptions): String =
+    JsNumber(this).toLocaleString(locales, options)
+
+fun Int.toLocaleString(locales: ReadonlyArray<JsString>): String =
+    JsNumber(this).toLocaleString(locales)
+
+fun Int.toLocaleString(locales: ReadonlyArray<JsString>, options: NumberFormatOptions): String =
```

Comment:
Please use similar formatting as for methods (2+ parameters = on separate lines)